openapi.yaml

```diff
@@ -1,279 +1,720 @@
+        meta:
+          $ref: '#/components/schemas/Meta'
+    ServicesIncluded:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/ServicesAttributes'
+        links:
+          $ref: '#/components/schemas/ServiceLinks'
+    LocationsResourceObject:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/Attributes'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInLocations'
+    ServiceSetResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesResourceObject'
+        meta:
+          $ref: '#/components/schemas/Meta'
+    ServicesResourceObject:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/ServicesAttributes'
+        links:
+          $ref: '#/components/schemas/ServiceLinks'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInServices'
+    Meta:
+      properties:
+        totalResults:
+          type: integer
+          description: Total number of results
+          example: 10
+        totalPages:
+          type: integer
+          description: Total number of pages
+          example: 10
+        currentPageNumber:
+          type: integer
+          description: Page number of the returned results
+          example: 1
+        currentPageSize:
+          type: integer
+          description: Number of results per page
+          example: 25
+    SelfLink:
+      properties:
+        self:
+          type: string
+          description: Url to this resource / location.
+    Attributes:
+      properties:
+        name:
+          type: string
+          description: Name of the location.
+        abbreviation:
+          type: string
+        latitude:
+          type: number
+          format: float
+        longitude:
+          type: number
+          format: float
+        geometry:
+          properties:
+            type:
+              type: string
+              enum:
+                - Polygon
+                - MultiPolygon
+            coordinates:
+              description: Array of ring arrays for each physical structure of a location
+              type: array
+              items:
+                description: Array of points that make up a ring.
+                type: array
+                items:
+                  type: array
+                  items:
+                    type: number
+                    format: float
+        summary:
+          type: string
+        description:
+          type: string
```

Comment:
noticing that `summary` and `description` do not have super clear functions in the example data I have. Same for `calendar`, which I wasn't able to find _any_ values for. I'm wondering if anyone can provide guidance about what these three attributes are supposed to be used for.

Reply:
Changing name of summary to zoneSummary

Reply:
Label description as something like "miscellaneous information about the building", and combine description and descriptionHTML into a single object.

Reply:
`calendar` is a URL to a calendar file for the building, but we're having trouble verifying that